### DISCUSSION TITLE
Webhooks respond empty data

### DISCUSSION THREAD

**Discussion by javeoff** [bukhalo/nestjs-telegraf#1211] [Q&A]
I am using telegraf-test library for e2e testing. It is my first experience, and I dont know which libraries may be better for that.
After including webhook to the server via constructor, my testing script successful sent signals to them and application ran methods with sending telegram messages to me. But in this case axios request (from telegraf-test) respond with empty data.

<img width="1440" alt="image" src="https://user-images.githubusercontent.com/62588539/192110182-aa682089-8631-426b-8593-10f06e252b5e.png">
<img width="565" alt="image" src="https://user-images.githubusercontent.com/62588539/192110402-30a7ca11-6c23-4c95-bb9b-8ecba5dea669.png">

**Comment by bukhalo** [comment]
I haven't used Telegraf Test for testing, but the webhooks work as they should. Maybe someone else can tell me how to integrate this library.
